### PR TITLE
Build order simplification

### PR DESCRIPTION
**Pull request by SchrodingersGat**
Simplifies the way that stock is allocated to a build order!

Fixes https://github.com/inventree/InvenTree/issues/1340

- [x] Update build documentation to match new approach

Documentation: https://github.com/inventree/inventree-docs/pull/104

### CODE REVIEW

File: InvenTree/build/templates/build/complete_output.html [hunk 1]
```diff
@@ -16,7 +17,7 @@ <h4>{% trans "Stock allocation is incomplete" %}</h4>
         <div class='panel panel-default'>
             <div class='panel panel-heading'>
                 <a data-toggle='collapse' href='#collapse-unallocated'>
-                    {{ unallocated_parts|length }} {% trans "parts have not been fully allocated" %}
+                    {{ unallocated_parts|length }} {% trans "tracked parts have not been fully allocated" %}
```
Comment (matmair) [inline InvenTree/build/templates/build/complete_output.html:20]: maybe use blocktrans here? as in PR1479

Comment (matmair) [reply InvenTree/build/templates/build/complete_output.html:20]: @SchrodingersGat

File: InvenTree/templates/js/build.js [hunk 2]
```diff
@@ -32,12 +32,17 @@ function newBuildOrder(options={}) {
...
     /* Generate action buttons for a build output.
      */
 
     var buildId = buildInfo.pk;
-    var outputId = output.pk;
+
+    if (output) {
+        outputId = output.pk;
+    } else {
+        outputId = 'untracked';
```
Comment (matmair) [inline InvenTree/templates/js/build.js:44]: Translation missing ;-)

**Review by matmair — COMMENTED** [review]
I see a lot of translations and that makes me happy 👼

File: InvenTree/templates/js/build.js (hunk 2 above)
Comment (SchrodingersGat) [reply InvenTree/templates/js/build.js:44]: The 'untracked' text here is an internal variable for JS - it is not user facing :)

**Pull request merged**